terraform-plans/configs/charm-nrpe_main.tfvars

```diff
@@ -0,0 +1,28 @@
+repository             = "charm-nrpe"
+repository_description = "A subordinate charm used to configure nrpe (Nagios Remote Plugin Executor)"
+branch                 = "main"
+templates = {
+  codeowners = {
+    source      = "./templates/github/CODEOWNERS.tftpl"
+    destination = ".github/CODEOWNERS"
+    vars        = {}
+  }
+  check = {
+    source      = "./templates/github/charm_check.yaml.tftpl"
+    destination = ".github/workflows/check.yaml"
+    vars        = {}
+  }
+  promote = {
+    source      = "./templates/github/charm_promote.yaml.tftpl"
+    destination = ".github/workflows/promote.yaml"
+    vars        = {}
+  }
+  release = {
+    source      = "./templates/github/charm_release.yaml.tftpl"
+    destination = ".github/workflows/release.yaml"
+    vars        = {
+      branch = "main",
+      runs_on = "[ubuntu-latest]",
```

Comment:
is it okay to not using arm64 runner?

Reply:
Yes this is fine - the same charm build runs on arm64 and amd64